### PR TITLE
[MM-19888]: Add documentation for silent installation of MSI

### PR DESCRIPTION
**Pull request by haardikdharma10**
#### Summary

Added documentation for silent installation of MSI.

#### Ticket Link

Fixes: #3220 
JIRA: https://mattermost.atlassian.net/browse/MM-19888

### CODE REVIEW

File: source/install/desktop-msi-gpo.rst [hunk 1]
```diff
@@ -120,3 +120,12 @@ Verify Group Policy settings in the installed Desktop app
...
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00018.png
+
+Silent Installation Guide
+--------------------------------------------
+
+Silent installation of MSI can be performed with the following command:
+
+``msiexec /i mattermost-desktop-v4.6.1-x64.msi /qn``
+
+Note: Current version is 4.6.1. In future, you may need to change it accordingly.
```
Comment (cwarnermm) [inline source/install/desktop-msi-gpo.rst:131]: Very minor update: "In **the** future, you may need to change it accordingly."

File: source/install/desktop-msi-gpo.rst [hunk 2]
```diff
@@ -120,3 +120,12 @@ Verify Group Policy settings in the installed Desktop app
 2. Verify the app loads the first server defined in the ``Edit group policy`` app.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00018.png
+
+Silent Installation Guide
+--------------------------------------------
+
+Silent installation of MSI can be performed with the following command:
```
Comment (cwarnermm) [inline source/install/desktop-msi-gpo.rst:127]: Consider changing this passive sentence to be active (for consistency with the remainder of this document).  Suggestion: "Perform a silent installation of MSI by running the following command:"

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Very minor changes suggested inline.

File: source/install/desktop-msi-gpo.rst (hunk 1 above)
Comment (haardikdharma10) [reply source/install/desktop-msi-gpo.rst]: Done! Thank you:)

**Comment by haardikdharma10** [comment]
Hi @cwarnermm, thank you for your time and suggestions. I have made the necessary changes.

**Review by cwarnermm — APPROVED** [review]
Looks good. Thank you, again, @haardikdharma10, for this excellent set of improvements to the documentation.

**Comment by jfrerich** [comment]
@deanwhillier, ping for review and welcome hope you had a great holidays

**Comment by deanwhillier** [comment]
Sincere apologies for not getting to this, I somehow missed seeing the review request. Will get on it asap.

**Review by Willyfrog — CHANGES_REQUESTED** [review]
as far as I know, MSI is still in beta and I'd mark it as such in the document, probably as a warning at the begining of the file.
other than that, looks good to me :)

**Review by Willyfrog — APPROVED** [review]
Nevermind my previous comment, it is stated at the top that the MSI is in beta, so no need to mention that twice

**Review by deanwhillier — APPROVED** [review]

**Comment by Willyfrog** [comment]
@metanerd I see other pages deploy correctly to github pages, but this one is failing, is there something we can do to fix it?

**Review by jaydeland — COMMENTED** [review]
Looks good

**Comment by jaydeland** [comment]
> @metanerd I see other pages deploy correctly to github pages, but this one is failing, is there something we can do to fix it?

I will have a look

**Comment by justinegeffen** [comment]
> @metanerd I see other pages deploy correctly to github pages, but this one is failing, is there something we can do to fix it?

This is because the PR is from a forked repo and the Github pages refers to our preview server which doesn't spin up previews from forked repos. :)

**Pull request merged**